### DISCUSSION TITLE
3D model does not render on Safari

### DISCUSSION THREAD

**Discussion by w-o-o-h** [google/model-viewer#3951] [Q&A]
It is about a month now since my models are not rendering in Safari anymore (both on MacOS and iOS).
Sample page [here](https://www.unspins.io/qoobs/quest/qq6/preview/099.html)
And full console with errors like:

<img width="623" alt="Screenshot 2022-11-20 at 17 09 33" src="https://user-images.githubusercontent.com/28869136/202909891-ccd98efb-823d-43d7-b5d3-9443ea5f6129.png">

Still, everything works with Chrome. Any ideas how to resolve this?

**Comment by elalish** [comment]
We're going to need a lot more information to be able to help. What change caused the problem? What version of MV do you use? What device(s)? What browser version(s)? For reference, your page works perfectly in Safari 16.1 on my Mac.

**Reply by w-o-o-h** [reply]
It appears to be related with my Safari version 14.1.2 but I have no idea when it got updated. Thanks for response!
